### PR TITLE
[E2E] [Picture] Update the picture

### PR DESCRIPTION
**Pull request by QuakeWang**
As the original images were in Chinese, now the English version has been added.

### CODE REVIEW

**Comment by zhongjiajie** [comment]
It is ready to go now? @QuakeWang

File: development/en-us/e2e-test.md [hunk 1]
```diff
@@ -104,7 +104,7 @@ The goToTab method is provided in SecurityPage to test the corresponding sidebar
     }
 ```
 
-![SecurityPage](../../img/e2e-test/SecurityPage.png)
+![SecurityPage](../../img/e2e-test/SecurityPage_en.png)
```
Comment (zhongjiajie) [inline development/en-us/e2e-test.md:107]: I think we should just add English version picture is better

Comment (QuakeWang) [reply development/en-us/e2e-test.md:107]: > I think we should just add English version picture is better

Do you mean that both Chinese and English choose to use the English version of the image?

Comment (zhongjiajie) [reply development/en-us/e2e-test.md:107]: yeah， thatwhat i mean

Comment (QuakeWang) [reply development/en-us/e2e-test.md]: > yeah， thatwhat i mean

done : )

**Review by zhongjiajie — APPROVED** [review]
LGTM, thanks for your contribution

**Pull request merged**